.github/workflows/deploy.yml

```diff
@@ -45,12 +45,6 @@ jobs:
           tar -zxvf ${{ inputs.artifact-name }}.tar.gz
           rm ${{ inputs.artifact-name }}.tar.gz
 
-      - name: Setup composer and install packages
```

Comment:
I don't understand what is happening here. Do we not need to setup composer as part of this workflow?

Reply:
So yes, but what was happening here was another container was being spun up and installing the packages and copying them back over for the archive. So this was working before because the default ubuntu PHP version was fine so when we ran composer on our own nothing broke. But I discovered that we needed to set the versions ourselves, so the steps after this one that was removed is setting up PHP and composer on the runner itself so we have the correct versions (and honestly this is how we should've done it from the beginning)